ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/identity_provider_config_service.proto

```diff
@@ -0,0 +1,142 @@
+service IdentityProviderConfigService {
+
+  // Create a new identity provider configuration.
+  // The request will fail if the maximum allowed number of separate configurations is reached.
+  rpc CreateIdentityProviderConfig (CreateIdentityProviderConfigRequest) returns (CreateIdentityProviderConfigResponse);
+
+  // Get the identity provider configuration data by id.
+  rpc GetIdentityProviderConfig (GetIdentityProviderConfigRequest) returns (GetIdentityProviderConfigResponse);
+
+  // Update selected modifiable attribute of a identity provider config resource described
+  // by the ``IdentityProviderConfig`` message.
+  rpc UpdateIdentityProviderConfig (UpdateIdentityProviderConfigRequest) returns (UpdateIdentityProviderConfigResponse);
+
+  // List all existing identity provider configurations.
+  rpc ListIdentityProviderConfigs (ListIdentityProviderConfigsRequest) returns (ListIdentityProviderConfigsResponse);
+
+  // Delete an existing identity provider configuration.
+  rpc DeleteIdentityProviderConfig (DeleteIdentityProviderConfigRequest) returns (DeleteIdentityProviderConfigResponse);
+
+}
+
+message IdentityProviderConfig {
+  // The identity provider identifier
+  // Must be a valid LedgerString (as describe in ``value.proto``).
+  // Required
+  string identity_provider_id = 1;
+
+  // When set, the callers using JWT tokens issued by this identity provider are denied all access
+  // to the Ledger API. Otherwise, the `sub` and `iss` claims provided in the JWT tokens will be 
+  // used to identify the users in the user management database and to grant access accordingly.
+  // Optional,
+  // Modifiable
+  bool is_deactivated = 2;
+
+  // Specifies the issuer of the JWT token that should be used in the "iss" claim of a JWT token to instruct the
+  // participant node to use this identity provider to authenticate the token.
+  // The issuer value is a case sensitive URL using the https scheme that contains scheme, host,
+  // and optionally, port number and path components and no query or fragment components.
+  // Required
+  // Modifiable
+  string issuer = 3;
+
+  // The JWKS (JSON Web Key Set) URL.
+  // The Ledger API will expect all tokens to be signed with RS256 (RSA Signature with SHA-256) with the public
+  // key loaded from the given URL.
+  // Required
+  // Modifiable
+  string jwks_url = 4;
+}
+
+
+message CreateIdentityProviderConfigRequest {
+  // Required
+  IdentityProviderConfig identity_provider_config = 1;
+}
+
+message CreateIdentityProviderConfigResponse {
+  IdentityProviderConfig identity_provider_config = 1;
+}
+
+message GetIdentityProviderConfigRequest {
+  // Required
+  string identity_provider_id = 1;
+}
+
+message GetIdentityProviderConfigResponse {
+  IdentityProviderConfig identity_provider_config = 1;
+}
+
+message ListIdentityProviderConfigsRequest {
+  // Pagination is not required for this request as an entire dataset can be retrieved in a single call.
```

Comment:
`// Pagination is not required as the resulting data set is small enough to be returned in a single call` ?